projects/cuelang.org/package.yml

```diff
@@ -0,0 +1,28 @@
+distributable:
+  url: https://github.com/cue-lang/cue/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: cue-lange/cue
+
+provides:
+  - bin/cue
+
+build:
+  script: |
+    go mod download
+    go build -v -ldflags="$LDFLAGS" "{{ prefix }}"/bin/cue ./cmd/cue
+  dependencies:
+    go.dev: ^1.18
+  env:
+    LDFLAGS:
+      - -s -w -X cuelang.org/go/cmd/cue/cmd.version=v{{ version }}
```

Comment:
cargo-culted from [brew](https://github.com/Homebrew/homebrew-core/blob/2b46e85bba6ce4615d13d4f018e938c1c623315a/Formula/cue.rb#L23).

Reply:
recommend splitting these up for better legibility and audits (and using the long-form flags if available)